osdep/io.c

```diff
@@ -692,6 +692,58 @@ off_t mp_lseek(int fd, off_t offset, int whence)
     return _lseeki64(fd, offset, whence);
 }
 
+void *mp_dlopen(const char *filename, int flag)
+{
+    wchar_t wfilename[MP_PATH_MAX];
+    
+    // convert filename (utf-8) to utf-16
+    int n = MultiByteToWideChar(
+        CP_UTF8,
+        MB_PRECOMPOSED,
+        filename,
+        -1,
+        wfilename,
+        MP_PATH_MAX);
+    
+    if (n == 0) {
+        return NULL;
+    }
+    
+    return (void *)LoadLibraryW(wfilename);
+}
+
+void *mp_dlsym(void *handle, const char *symbol)
+{
+  return (void *)GetProcAddress((HMODULE)handle, symbol);
+}
+
+static char mp_dlerror_buffer[250];
```

Comment:
Right, it seems dlerror() can unfortunately not be implemented without global state. But it should use TLS, try adding _Thread_local to the declaration.

Reply:
Okay, it compiles correctly here